README.md

```diff
@@ -1,4 +1,4 @@
 # hello-world
 Github tutorial on creating a depository - my first one.
 
-As I am a lone cowhand from the Rio Grande, this is probably lok very simple.
```

Comment:
As I am a lone cowhand from the Rio Grande, this is probably going to look very simple.